crates/aptos-jwk-consensus/src/jwk_manager/mod.rs

```diff
@@ -103,15 +103,27 @@ impl JWKManager {
             .unwrap_or_default()
             .into_provider_vec()
             .into_iter()
-            .map(|provider| {
-                JWKObserver::spawn(
-                    self.epoch_state.epoch,
-                    self.my_addr,
-                    provider.name.clone(),
-                    provider.config_url.clone(),
-                    Duration::from_secs(10),
-                    local_observation_tx.clone(),
-                )
+            .flat_map(|provider| {
```

Comment:
Why the `flat_map`? You are mapping `provider` to one value, not to an iterator over values no?

Reply:
looks like i meant `filter_map` 🤦